dashboard/app/models/peer_review.rb

```diff
@@ -142,9 +142,25 @@ def self.create_for_submission(user_level, level_source_id)
     end
   end
 
+  def self.get_review_completion_status(user, script)
+    if user &&
+        script.peer_reviews_to_complete.try(:>, 0) &&
+        Plc::EnrollmentUnitAssignment.exists?(user: user, plc_course_unit: script.plc_course_unit)
+      reviews_done = PeerReview.where(reviewer: user, script: script, status: PeerReview.statuses.values).size
```

Comment:
Makes the testing harder with the chained stubs that I would have to do. I'd rather just stub one where method